src/vs/editor/contrib/smartSelect/smartSelect.ts

```diff
@@ -165,7 +165,10 @@ class GrowSelectionAction extends AbstractSmartSelect {
 			kbOpts: {
 				kbExpr: EditorContextKeys.editorTextFocus,
 				primary: KeyMod.Shift | KeyMod.Alt | KeyCode.RightArrow,
-				mac: { primary: KeyMod.CtrlCmd | KeyMod.WinCtrl | KeyMod.Shift | KeyCode.RightArrow },
+				mac: {
+					primary: KeyMod.CtrlCmd | KeyMod.WinCtrl | KeyMod.Shift | KeyCode.RightArrow,
+					secondary: [KeyMod.WinCtrl | KeyMod.Shift | KeyCode.RightArrow],
+				},
```

Comment:
This is only for mac, right? How about the other platforms?

Reply:
On other platforms, they are good enough: the primary shortcut is KeyMod.Shift | KeyMod.Alt | KeyCode.LeftArrow, which is the same length as Mac.

Reply:
Ah, makes sense. Thanks for clarifying